src/popup/component/Button/index.js

```diff
@@ -23,9 +40,7 @@ export default class Button extends Component {
         switch (buttonType) {
             case BUTTON_TYPE_COMMON_BUTTON:
             default:
-                currentClassName = cx("click-cursor",{
-                    "common-button": true,
-                    [propsClass]: true,
+                currentClassName = cx("common-button", propsClass, {
```

Comment:
https://www.npmjs.com/package/classnames
```ts
classNames('foo', { bar: true, duck: false }, 'baz', { quux: true }); // => 'foo bar baz quux'
classNames(null, false, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
```

In case of our disabled button:
- original

  ```ts
  cx("click-cursor", {
    "common-button": true, 
    ["account-common-btn account-common-btn-danger"]: true, 
    "click-cursor": false, 
    "click-cursor-disable": true,
  })
  // => "click-cursor common-button account-common-btn account-common-btn-danger click-cursor-disable"
  ```

- removed needless unconditional `click-cursor`. CSS context

  ```css
  .click-cursor { cursor: pointer; }
  .click-cursor-disable { cursor: not-allowed; }
  ```

  ```ts
  cx({
    "common-button": true, 
    ["account-common-btn account-common-btn-danger"]: true, 
    "click-cursor": false, 
    "click-cursor-disable": true,
  })
  // => "common-button account-common-btn account-common-btn-danger click-cursor-disable"
  ```

- used unconditional classes like a normal person (and typescript didn't like newly-typed `propsClass: string | undefined` as an object key)

  ```ts
  cx("common-button", "account-common-btn account-common-btn-danger", {
    "click-cursor": false, 
    "click-cursor-disable": true,
  })
  // => "common-button account-common-btn account-common-btn-danger click-cursor-disable"
  ```

Reply:
ok yeah that old one was really weird. probably written without an understanding of `cx`.